page-objects/pages/TASKListPage.js

```diff
@@ -154,19 +153,47 @@ async function clickDeleteTaskButtonInPopup() {
 	await driver.refresh();
 }
 
-async function clickAtTask(){
+async function clickAtTask(taskName) {
```

Comment:
Do we need both clickOnTask(button) | clickAtTask(taskName) or can we maybe combine them? Can be done as follow up.

Reply:
Will be done within BC-1008.